docs/_docs/Applicants/Application-process.md

```diff
@@ -73,101 +73,85 @@ or at any time during the course, I will be expelled from the Academy.
-new concepts. In the exercises, we'll evaluate if you understood these new concepts. Only the exercises need to
-be submitted and will be graded. There are no time constraints for the Learning Units: you can start them at any
-time after 2022-07-11 and you can make pauses while working on them, as long as you submit them before 2022-07-24.
+The Learning Units about pandas have two parts: the learning part and the exercises. In the learning part, you will learn
+new concepts. In the exercises, we will evaluate if you understood these new concepts. Only the exercises need to
+be submitted and will be graded. You can complete the Learning units at any
+time between 23-10-2023 and 05-11-2023 and you can make pauses while working on them.
 
-The dates when you submit the coding test and the Learning Units don't have any influence on the selection
-process, as long as it's before 2022-07-24. Finishing earlier doesn't give you any advantage, so take your time!
+The dates of the submission of the coding test and of the Learning Units don't have any influence on the selection
+process, as long as it's before 05-11-2023 23h59 UTC. Finishing earlier doesn't give you any advantage.
 
-These are the main topics that will be evaluated on the coding test:
+These are the main topics that will be evaluated in the coding test:
 - Programming in Python
 - Numpy
 - Basics of working with the command line terminal
 - Git
 
 You can see them in more detail [here](https://docs.google.com/spreadsheets/d/1JzRfuacM5Q_XsnSMgEqOZH_cCBnT0J6K-e-687fFa6s/edit#gid=2018895387).
 
-This is the content that will be taught and evaluated on the Learning Units:
+This is the content that will be taught and evaluated in the Learning Units:
 - [Pandas 101](https://github.com/LDSSA/curriculum-development/blob/master/curriculum/01-bootcamp-and-binary-classification.md#slu01)
 - [Subsetting Data in Pandas](https://github.com/LDSSA/curriculum-development/blob/master/curriculum/01-bootcamp-and-binary-classification.md#slu02)
 - [Visualization with Pandas & Matplotlib](https://github.com/LDSSA/curriculum-development/blob/master/curriculum/01-bootcamp-and-binary-classification.md#slu03)
 
 
 ### Candidates selection
 
-After the admission tests period, we'll select the Academy students by taking a random sample over all the candidates
+After the admission test period, we will select the Academy students by taking a random sample over all the candidates
 who passed the tests. We'll make sure that the [entrance restrictions](#entrance-type-restrictions) are respected.
 
 The candidates who are selected will confirm their application by paying the admission fee. If this fee is
 not paid on time, we'll reject the candidate and give their seat to the next candidate in line.
 
 ### Fees
 
-This year, because of the Covid-19 crisis, we've kept the lowered fees from 2020/2021.
+This year, we are going back to the fees from before the Covid-19 crisis.
```

Comment:
I would remove this mention. If someone asks, we can answer that these are the normal price and we only lowered the value for the Covid period.

Reply:
OK.